src/seedu/addressbook/commands/ClearCommand.java

```diff
@@ -13,7 +13,6 @@
 
     public ClearCommand() {}
 
-
```

Comment:
Try to avoid unnecessary changes. Though it is simple, it is not a good practice to ignore it